.bumpversion.cfg

```diff
@@ -0,0 +1,10 @@
+[bumpversion]
+current_version = 0.1.1
```

Comment:
If we decided to decouple the `gt4py` and `eve` version numbers, would that be supported without changing the project structure? It seems to me there is only one single source of truth per repository supported by bumpversion.

Is it possible to make a `.bumpversion.gt4py.cfg` and a `.bumpversion.eve.cfg` and tell the CLI which to use? Or would we have one `.bumpversion.cfg` in `src/eve` and another one in `src/gt4py`?

Reply:
Good point. Decoupling eve and gt4py would only happen after changing the structure of the repo in such a way that eve, gt4py and other projects are folders at the root level of the repo. In this case, every project could have its own `.bumpversion.cfg` file. Anyway, the CLI tool is quite flexible and accepts different config files using the option `--config-file`

Reply:
👍